Code/Legacy/CrySystem/SystemInit.cpp

```diff
@@ -1469,7 +1469,11 @@ void CSystem::CreateSystemVars()
     REGISTER_STRING_CB("g_language", "", VF_NULL, "Defines which language pak is loaded", CSystem::OnLanguageCVarChanged);
 
     // adding CVAR to toggle assert verbosity level
+#if !defined(CARBONATED)
     const int defaultAssertValue = 1;
+#else
+    const int defaultAssertValue = Legacy::System::CVars::sys_asserts;
+#endif
```

Comment:
Change this around to 
#if defined(CARBONATED)
..
#else
old code
#endif

(to stay consistent with existing CARBONATED guards)

Reply:
Updated according to this comment